include/swift/AST/Attr.def

```diff
@@ -322,6 +322,9 @@ DECL_ATTR(differentiable, Differentiable,
           OnFunc | LongAttribute | NotSerialized,
           /* Not serialized */ 76)
 
+SIMPLE_DECL_ATTR(compilerEvaluable, CompilerEvaluable,
+                 OnFunc | OnConstructor, 77)
```

Comment:
Is it serialized? If not, add a comment.

Reply:
Is serializing an attribute something that I need to add extra code for somewhere? I didn't pass the `NotSerialized` flag.

Reply:
No need for extra code for now.